react/src/router/router.ts

```diff
@@ -0,0 +1 @@
+
```

Comment:
del

Reply:
coversation: Del router.ts. WIP - work in progress
https://github.com/Kira-Vishn9/AdminAudio/pull/2#discussion_r1252923682

Commit example

Reply:
done